client/my-sites/sidebar/index.jsx

```diff
@@ -603,6 +605,11 @@ export class MySitesSidebar extends Component {
 			);
 		}
 
+		// Don't show the "Plans" sidebar item in WP Mobile App WebViews to avoid them being rejected
```

Comment:
> "to avoid them being rejected"

thanks for putting in this note - the explanatory comment is gold

something more specific could provide a stronger clue to what exactly is rejected.

```ts
// Hide "Plans" because the App Stores won't allow us to show purchase options that aren't in-app purchases
```

something like that